src/main/groovy/org/transmartproject/batch/support/StringUtils.groovy

```diff
@@ -0,0 +1,11 @@
+package org.transmartproject.batch.support
+
+/**
+ * Misc string utils
+ */
+class StringUtils {
+
+    static String escapeForLike(String s, char escapeChar = '\\') {
+        s.replaceAll(/[\\_%]/, "$escapeChar\\\$0")
+    }
```

Comment:
Fixed in https://github.com/thehyve/transmart-batch/commit/393004bbea53e1acfa55e0ed8e1160e455eeaafe